Formula/gopass-jsonapi.rb

```diff
@@ -0,0 +1,51 @@
+class GopassJsonapi < Formula
+  desc "Gopass Browser Bindings"
+  homepage "https://github.com/gopasspw/gopass-jsonapi"
+  url "https://github.com/gopasspw/gopass-jsonapi/archive/v1.11.1.tar.gz"
+  sha256 "3c1666cdbf78a73736b089c3188c06724de53a96e43cb1d82fedd7e9543c120f"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "gopass"
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.version=#{version}"
+  end
+
+  def caveats
+    "You need to run gopass-jsonapi configure for each browser you want to use with gopass."
+  end
```

Comment:
This is already documented upstream, so there's no need to include this. `caveats` are for Homebrew-specific messages.

Reply:
Thanks. I have removed the `caveats` block.